ipynb/tests/Generic_EAS_Tests.ipynb

```diff
@@ -0,0 +1,533 @@
+      "2016-12-21 15:44:33,000 INFO    : root         :   /home/brejac01/sources/lisa/logging.conf\n"
+     ]
+    }
+   ],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import logging\n",
+    "from conf import LisaLogging\n",
+    "LisaLogging.setup()\n",
+    "logging.getLogger().setLevel(logging.ERROR) # ixnay on the ogginglay\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "from perf_analysis import PerfAnalysis\n",
+    "\n",
+    "import trappy\n",
+    "from trappy import ILinePlot\n",
+    "from trappy.stats.grammar import Parser"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run test workload\n",
+    "\n",
+    "See `tests/eas/generic.py` for details of the available tests. To run different tests, simply change the constructor call that assigns to `t` and then run the remaining cells below. \n",
+    "\n",
+    "If you simply want to run all the tests and get pass/fail results, use this command in the LISA shell: `lisa-test tests/eas/generic.py`. This notebook is intended as a starting point for analysing what scheduler behaviour caused a failure.\n",
+    "\n",
+    "Target configuration is taken from `$LISA_HOME/target.config` - you'll need to edit that file to provide connection details for the target you want to test."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from tests.eas.generic import TwoBigTasks, TwoBigThreeSmall, RampUp, RampDown, EnergyModelWakeMigration, OneSmallTask"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "    Test EAS for tasks alternating beetween 10% and 50%\n",
+      "    \n"
+     ]
+    }
+   ],
+   "source": [
+    "t = EnergyModelWakeMigration(methodName=\"test_task_placement\")\n",
+    "print t.__doc__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "t.setUpClass()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "experiment = t.executor.experiments[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examine trace\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "utils = t.get_task_utils_df(experiment)"
```

Comment:
Sorry! I did not make myself clear. I did not mean to say that the calls don't return anything but that you can't see anything being printed as a result of them being run. I suppose printing the docstrings could work but what I meant to say is that it would be nice to have a section where you can see more details on the test being run. Details on the workload, the data frames obtained, something that could represent the basis of a debugging process.